addons/web/static/src/views/fields/badge_selection/badge_selection_field.js

```diff
@@ -63,6 +65,38 @@ export class BadgeSelectionField extends Component {
         return JSON.stringify(value);
     }
 
+    get spanClass() {
+        if (this.props.colorField && this.props.record.data[this.props.name]) {
+            return `badge rounded-pill o_badge_color_${
+                this.props.record.data[this.props.colorField]
+            }`;
+        }
+        return "";
+    }
+
+    badgeSelectionClass(value) {
+        const badgeClass = [];
+        if (this.props.size === "sm") {
+            badgeClass.push("btn-sm");
+        }
+        if (this.props.size === "lg") {
+            badgeClass.push("btn-lg");
+        }
```

Comment:
Equivalent to: 
```suggestion
        if (["sm", "lg"].includes(this.props.size)) {
            badgeClass.push(`btn-{this.props.size}`);
        }
```

😇